src/bandit-evaluator.ts

```diff
@@ -36,7 +41,13 @@ export class BanditEvaluator {
       banditModel.actionProbabilityFloor,
     );
     const selectedActionKey: string = this.selectAction(flagKey, subjectKey, actionWeights);
-    const optimalityGap = 0; // TODO: compute difference between selected and max
+
+    // Compute optimality gap in terms of score
```

Comment:
For now we want optimality gap in terms of score (as opposed to percentage).

🗨️ Relevant slack conversation: ["...should optimality gap be in terms of percentage (e.g., weighted) than absolute score?..."](https://eppo-group.slack.com/archives/C05R6BK5BB9/p1718232691365099)